apps/server/lib/http/facades/action.ts

```diff
@@ -35,7 +35,7 @@ export class ActionFacade {
 
 	async processAction(
 		context,
-		sessionToken,
+		sessionId,
```

Comment:
If we're going to change the variable name, I'd rather we go with `session` instead to match the rest of our code base. Do a quick grep under `@balena/jellyfish-core/lib` for example and you'll see what I mean.

Reply:
I thought `sessionId` would be more discriminative/easier to follow as we also have session objects sometimes, but you do have a good point. Will update that.

Reply:
`sessionId` was replaced with `session`